gapis/api/vulkan/api/vk12structs.api

```diff
@@ -260,3 +260,132 @@ class VkSubpassDescriptionDepthStencilResolve {
     VkResolveModeFlagBits            stencilResolveMode
     const VkAttachmentReference2*    pDepthStencilResolveAttachment
 }
+
+class VkDescriptorSetVariableDescriptorCountAllocateInfo {
+    VkStructureType sType
+    const void*     pNext
+    u32             descriptorSetCount
+    const u32*      pDescriptorCounts
+}
+
+class VkDescriptorSetLayoutBindingFlagsCreateInfo {
+    VkStructureType                 sType
+    const void*                     pNext
+    u32                             bindingCount
+    const VkDescriptorBindingFlags* pBindingFlags
+}
+
+@internal
+class DescriptorSetLayoutBindingFlagsCreateInfo {
```

Comment:
Drop the `CreateInfo` part of the name.

Reply:
Wouldn't it be confusing with bitmask `DescriptorSetLayoutBindingFlags`

Reply:
you mean `VkDescriptorSetLayoutBindingFlags`, I presume? I mean, the confusion is no more between `VkDescriptorSetLayoutBindingFlags` and `DescriptorSetLayoutBindingFlags` as is between `VkDescriptorSetLayoutBindingFlagsCreateInfo` and `DescriptorSetLayoutBindingFlags`, no? Looking at the rest of the api files, I suppose there isn't really that consistent of a pattern, but a somewhat broadly used pattern would be `DescriptorSetLayoutBindingFlagsInfo`, let's go with that one then.

Reply:
`DescriptorSetLayoutBindingFlagsCreateInfo` is not a great name, but it is consistent with the analogous vulkan struct
 `VkDescriptorSetLayoutBindingFlagsCreateInfo`. I can drop the `Create` part to make it less wordy if there is a strong preference.

Reply:
I see the pattern of `FooCreateInfo` becomming `FooInfo` in other places, so let's follow that.